documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,405 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Migration from OpenStack moves to being a fully supported feature
```

Comment:
this item should drop (OpenStack is a fully supported feature in MTV 2.5)

Reply:
removed